components/GreetingButton.js

```diff
@@ -1,7 +1,7 @@
 import { View, Text, StyleSheet, TouchableHighlight } from 'react-native';
 
 
-const GreetingButton = ({mantras}) => {
+const GreetingButton = ({ quote, fetch}) => {
```

Comment:
Passing params in here... I believe I'm following how this is working, but would love to go through a code review together for understanding :)